### ISSUE TITLE
English alt text

### ISSUE THREAD

**Issue by xfq** [w3c/i18n-drafts#775]
https://github.com/w3c/i18n-drafts/blob/b617ae5fad9821329e61a9cdc3629e0577eecdd1/articles/definitions-characters/index.en.html#L145

and

https://github.com/w3c/i18n-drafts/blob/b617ae5fad9821329e61a9cdc3629e0577eecdd1/articles/definitions-characters/index.en.html#L166

The alt text should be English instead of French.

**Comment by xfq** [comment]
I looked at it again and found that the SVG [image](https://developer.mozilla.org/en-US/docs/Web/SVG/Reference/Element/image) element does not have an `alt` attribute at all. We should use the [title](https://svgwg.org/svg2-draft/struct.html#TitleElement) element.

**Comment by xfq** [comment]
The section is here: https://www.w3.org/International/articles/definitions-characters/index.en.html#charsets

**Comment by xfq** [comment]
Fixed in https://github.com/w3c/i18n-drafts/commit/5540ce5d365fa25a3b84be041f58de9cddf1e35b
